resources/ssm_parameter_store.rb

```diff
@@ -0,0 +1,124 @@
+property :name,                        String, required: true
```

Comment:
You don't want to define the name property. Chef defines it for you already and it actually has a few more options out of the box